src/lib.rs

```diff
@@ -92,3 +92,13 @@ pub use crate::hasher_impl::sha3::{Keccak224, Keccak256, Keccak384, Keccak512};
 pub use crate::hasher_impl::sha3::{Sha3_224, Sha3_256, Sha3_384, Sha3_512};
 #[cfg(feature = "strobe")]
 pub use crate::hasher_impl::strobe::{Strobe256, Strobe512, StrobeHasher};
+
+/// Code reserved for Identity "hash"
+pub const IDENTITY_CODE: u64 = 0x0;
```

Comment:
This doesn't necessarily need to be public, can we make it crate-private?

Reply:
its still useful imo, e.g. `Multihash::wrap(multihash::IDENTITY_CODE, &bytes)`

Reply:
I see that. I am just wary about extending the public API with new items that are not essential to be there. If we want to discourage the use of the identity hasher, it seems reasonable to have users declare that constant by themselves if they actually want to use it.